packages/react-chat/src/components/Form/Menu.tsx

```diff
@@ -0,0 +1,66 @@
+import React from "react";
+import styled from "styled-components";
+
+import { useContextMenu } from "../../hooks/useContextMenu";
+import { copyImg } from "../../utils/copyImg";
+import { downloadImg } from "../../utils/downloadImg";
+import { CopyIcon } from "../Icons/CopyIcon";
+import { DownloadIcon } from "../Icons/DownloadIcon";
+import { textSmallStyles } from "../Text";
+
+interface MenuProps {
+  image: string;
+}
+
+export const Menu = ({ image }: MenuProps) => {
```

Comment:
```suggestion
export const ImageMenu = ({ imageId }: MenuProps) => {
```

This element is specifically for image so I think name should be changed according to that also we need image id so prop should be named like that